tests/transformers/framework_fill_na_transformer/v1/test_framework_fill_na_transformer.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+from pyspark.sql import SparkSession, DataFrame
+from spark_pipeline_framework.transformers.framework_csv_loader.v1.framework_csv_loader import (
+    FrameworkCsvLoader,
+)
+from spark_pipeline_framework.utilities.spark_data_frame_helpers import (
+    create_empty_dataframe,
+)
+from spark_pipeline_framework.transformers.framework_fill_na_transformer.v1.framework_fill_na_transformer import (
+    FrameworkFillNaTransformer,
+)
+
+
+def test_framework_fill_na_transformer(spark_session: SparkSession) -> None:
+    # create a dataframe with the test data
+    data_dir: Path = Path(__file__).parent.joinpath("./")
+
+    df: DataFrame = create_empty_dataframe(spark_session=spark_session)
+
+    view: str = "primary_care_protocol"
+    FrameworkCsvLoader(
+        view=view,
+        path_to_csv=data_dir.joinpath("primary_care_protocol.csv"),
+        clean_column_names=False,
+    ).transform(df)
+
+    # ensure we have all the rows even the ones we want to drop
+    result_df: DataFrame = spark_session.table(view)
+    result_df = result_df.withColumn(
+        "Minimum Age", result_df["Minimum Age"].cast("float")
+    )
+    print(result_df.dtypes)
+    result_df.createOrReplaceTempView(view)
+    assert 7 == result_df.count()
+
+    # drop the rows with null NPI or null Last Name
+
+    FrameworkFillNaTransformer(
+        replacement_value=1.0, columns_to_check=["Minimum Age"], view=view
+    ).transform(df)
+
+    # assert we get only the rows with a populated NPI
+    result_df = spark_session.table(view)
+    print(result_df.agg({"Minimum Age": "sum"}).collect()[0][0])
+    assert 7 == result_df.count()
```

Comment:
Can we make sure we have a test that includes more than one column for test coverage?